templates/full/src/controls/ui.tsx

```diff
@@ -129,6 +134,20 @@ function WebViewExample() {
   )
 }
 
+function VideoExample() {
+  return (
+    <>
+      <StackScreen title="Video" />
+      <Video
+        source="https://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4"
```

Comment:
I took video from Expo documentation so no need to host it. We could consider uploading our own!

Reply:
yeah this is fine, its a VERY common example video